checkout/v3/mac/index.md

```diff
@@ -4,30 +4,18 @@ title: Introduction
 estimated_read: 3
 hide_from_sidebar: false
 description: |
-  **MAC** is the Checkout option for merchants with their own consumer
+  **MAC** is the Checkout for merchants with their own consumer
```

Comment:
I think it's wise to spell out `MAC` whenever there's room for it, so we avoid the confusion with Apple Macintosh as much as possible.

```suggestion
  **Merchant Authenticated Consumer** is the Checkout for merchants with their own consumer
```